configuration/configuration.go

```diff
@@ -156,6 +156,14 @@ func LoadConfiguration() (*Configuration, error) {
 		config.GenesisBlockIdentifier = ethereum.GoerliGenesisBlockIdentifier
 		config.Params = params.GoerliChainConfig
 		config.GethArguments = ethereum.GoerliGethArguments
+	case Testnet:
+		config.Network = &types.NetworkIdentifier{
+			Blockchain: ethereum.Blockchain,
+			Network:    ethereum.DevNetwork,
+		}
+		config.GenesisBlockIdentifier = nil
+		config.Params = params.AllCliqueProtocolChanges
```

Comment:
do we this change?

Reply:
looks to me like the config for the dev testnet: https://github.com/ethereum/go-ethereum/blob/master/params/config.go#L280

it has the right chain id 1337, all the protocol transitions are set to 0 and uses clique consensus.